docs/src/main/sphinx/release/release-411.md

```diff
@@ -122,17 +122,17 @@
 ## Oracle connector
 
 * Improve performance of queries when the network latency between Trino and
-  Oracle is high, or when selecting or when selecting a small number of rows. ({issue}`16644`)
+  Oracle is high, or when selecting a small number of rows. ({issue}`16644`)
```

Comment:
```suggestion
  Oracle is high, or when selecting a small number of columns. ({issue}`16644`)
```

Reply:
Same for the others. I checked the original issue and this is the correct wording. cc @colebow